source/fileUtils.py

```diff
@@ -0,0 +1,24 @@
+#fileUtils.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+import os
+import ctypes
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+from logHandler import log
+
+@contextmanager
+def FaultTolerantFile(name):
```

Comment:
Please add a docstring with details on what this is/does and maybe example usage. The Python term for one of these `with` things is "context manager".